packages/components/src/slot-fill/bubbles-virtually/use-slot.ts

```diff
@@ -24,10 +24,18 @@ export default function useSlot( name ) {
 
 	const api = useMemo(
 		() => ( {
-			updateSlot: ( fillProps ) => registry.updateSlot( name, fillProps ),
-			unregisterSlot: ( ref ) => registry.unregisterSlot( name, ref ),
-			registerFill: ( ref ) => registry.registerFill( name, ref ),
-			unregisterFill: ( ref ) => registry.unregisterFill( name, ref ),
+			updateSlot: (
+				fillProps: Parameters< typeof registry.updateSlot >[ 1 ]
+			) => registry.updateSlot( name, fillProps ),
+			unregisterSlot: (
+				ref: Parameters< typeof registry.unregisterSlot >[ 1 ]
```

Comment:
I wonder if we can be specific here rather than being so verbose. Or at least infer the types in a more general way that doesn't require us to specify them for each of the properties in this object.